java/src/jmri/jmrix/can/adapters/loopback/ConnectionConfig.java

```diff
@@ -24,21 +25,38 @@ public ConnectionConfig(jmri.jmrix.SerialPortAdapter p) {
 
     // Needed for instantiation by reflection, do not remove.
     /**
-     * Create a connection configuration without a preexisting adapter.
+     * Ctor for a connection configuration with no preexisting adapter.
+     * {@link setInstance()} will fill the adapter member.
      */
     public ConnectionConfig() {
         super();
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void loadDetails(JPanel details) {
+        super.loadDetails(details);
+        portBoxLabel.setVisible(false); // hide portBoxLabel during config, cf. as when extending jmri.jmrix.AbstractSimulatorConnectionConfig
```

Comment:
Hides port and baud labels + boxes in the connection config pane Options, as they're not used but Loopback is not extending the AbstractSimulatorConnectionConfig but SerialConnectionConfig